docs/LLS_Metallicities-Wotta/README_python_get_pdf_cdf.py

```diff
@@ -0,0 +1,196 @@
+##Example: How to get walkers or create a PDF and CDF
```

Comment:
If this is meant as a tutorial of sorts,
can we make it a Notebook instead?

I should prefer that for stuff within docs/